Tiltfile

```
@@ -39,8 +39,10 @@ IMG = 'controller:latest'
 CONTROLLERGEN = 'rbac:roleName=manager-role crd webhook paths="./..." output:crd:artifacts:config=config/crd/bases;'
 DISABLE_SECURITY_CONTEXT = True
 
+
 def yaml():
-    data = local('cd config/manager; kustomize edit set image controller=' + IMG + '; cd ../..; kustomize build config/manager')
+    data = local('cd config/manager; kustomize edit set image controller=' +
+                 IMG + '; cd ../..; kustomize build config/manager')
```

nit, this looks like python.
```suggestion
    data = local(f'cd config/manager; kustomize edit set image controller={IMG}; cd ../..; kustomize build config/manager')
```